src/Factories/FunFunFactory.php

```diff
@@ -978,24 +1001,38 @@ private function getSofortClient(): SofortClient {
 		} );
 	}
 
-	private function newCreditCardUrlGenerator(): CreditCardUrlGenerator {
-		return new CreditCardUrlGenerator( $this->newCreditCardUrlConfig() );
-	}
-
 	private function newCreditCardUrlConfig(): CreditCardConfig {
 		$locale = \Locale::parseLocale( $this->getLocale() );
-		return CreditCardConfig::newFromConfig( array_merge( $this->config['creditcard'], [ 'locale' => $locale['language'] ] ) );
+		return CreditCardConfig::newFromConfig(
+			array_merge( $this->config['creditcard'], [ 'locale' => $locale['language'] ] ),
+			new TranslatableDescriptionStub()
```

Comment:
Please pass in `credit_card_item_name_donation`, which is especially important for credit cards since it contains an interval-dependent placeholder